prover/lang/kore-lang.md

```diff
@@ -778,6 +797,13 @@ Simplifications
   rule hasImplicationContext(\typeof(P, _))
     => hasImplicationContext(P)
 
+  syntax Pattern ::= "#\\exists" "{" Patterns "}" Pattern [function]
```

Comment:
This needs a better name. There are already functions called `#exists`, `#not` that propagate these constructs into patterns.